tests/nanoemoji_test.py

```diff
@@ -38,14 +40,26 @@
 )
 
 
-@pytest.fixture(scope="module", autouse=True)
+# https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
```

Comment:
That's fancy!
Alternatively, we could also simply switch to use the default `tmp_path` pytest fixture. It keeps temporary directories from the previous 3 test runs and removes the older ones on a rolling basis.
I refrained from using that to reduce the amount of diffs.
https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
I tried leaving temp files for an entire test run and it's quite a lot so for now I prefer the "silent when successful" model.